cmd/tmsp-cli/tmsp-cli.go

```diff
@@ -15,10 +16,47 @@ import (
 	"github.com/urfave/cli"
 )
 
+//structure for data passed to print response
+// variables must be exposed for JSON to read
+type pr struct {
```

Comment:
not sure this is a good naming

p.s. here is a nice presentation from google dev https://talks.golang.org/2014/names.slide#1